columbo/_interaction.py

```diff
@@ -181,14 +189,22 @@ def __init__(
         :param cli_help: Optional help message to be displayed for command line interface.
         :param should_ask: If `None`, the question is asked. Otherwise, the callable will be passed the answers that
             have been provided this far and should return `True` if the question should be asked.
+        :param value_if_not_asked: If provided and if should_ask is being used, this value will be recorded as an answer if should_ask evaluates to False.
         """
-        super().__init__(name, message, cli_help, should_ask)
+        super().__init__(
+            name, message, cli_help, should_ask, value_if_not_asked=value_if_not_asked
```

Comment:
Style nit: for consistency and readability, I think `cli_help` and `should_ask` should be used as kwargs as they are defined in the init function signature, just as you're doing with `value_if_not_asked`.